pkgs/by-name/au/authentik/package.nix

```diff
@@ -110,75 +110,6 @@ let
   python = python312.override {
     self = python;
     packageOverrides = final: prev: {
-      django-tenants = prev.buildPythonPackage rec {
-        pname = "django-tenants";
-        version = "unstable-2024-01-11";
-        src = fetchFromGitHub {
-          owner = "rissson";
```

Comment:
Authentik still needs to use the fork afterwards

Reply:
Updated to make use of the fork